1-mandatory/1-writers.js

```diff
@@ -62,7 +64,12 @@ Exercise 2:
 */
 
 function logDeadWritersInTheirForties() {
-  // write your code here
+  for (author of writers){
+  if (author.age >= 40 && author.age <=49 && author.alive === false){
+    console.log(`Writer ${author.firstName} ${author.lastName} died at ${author.age} years old.`)
+  }
+  }
+  
 }
```

Comment:
Comment: Well done

Consider: Here is another way to do it.
function logAllWriters() {
// write your code to log all writers here
for (let i = 0; i < writers.length; i++) {
const writer = writers[i];
console.log(Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.);
}
}